packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -102,10 +89,16 @@ const CompleteResetPasswordContainer = ({
         replace: true,
         state: {
           email,
-          fancyBannerSuccessMessage: {
-            heading: successBannerMessageHeading,
-            message: successBannerMessageMessage,
-          },
+          successBanner: {
+            localizedSuccessBannerHeading: ftlMsgResolver.getMsg(
+              'reset-password-complete-banner-heading',
+              'Your password has been reset.'
+            ),
+            localizedSuccessBannerDescription: ftlMsgResolver.getMsg(
+              'reset-password-complete-banner-message',
+              'Don’t forget to generate a new account recovery key from your Mozilla account settings to prevent future sign-in issues.'
+            ),
+          } as LocationState,
```

Comment:
Do we have to type cast this?

Reply:
We don't _have_ to cast it, but it helped me spot errors when I initially misnamed one of the keys.